modules/applications/pages/ui-builder/common-apis-to-use.adoc

```diff
@@ -0,0 +1,101 @@
+= Common APIs to use
+:page-aliases: applications:common-apis-to-use.adoc
```

Comment:
```suggestion
```

As you added this page in this version, you don't need to add the :page-aliases: attribute. For more details: https://docs.antora.org/antora/latest/page/page-aliases/